uportal-war/src/main/java/org/apereo/portal/portlets/lookup/PersonLookupHelperImpl.java

```diff
@@ -427,9 +427,9 @@ protected IAuthorizationPrincipal getPrincipalForUser(final IPerson person) {
      * @param attributeNames
      * @return
      */
-    protected Set<String> getAvailableAttributes(
+    protected Set<String> getPermittedAttributes(
```

Comment:
Would love to see a unit test or three, verifying no regression and addition of the new complexity.